src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -135,33 +135,6 @@ public AppiumDriver(AppiumLocalService service, ICapabilities appiumOptions, Tim
 
         #region MJsonMethod Members
 
-        /// <summary>
-        /// Rotates the device.
-        /// </summary>
-        /// <param name="opts">
-        /// Rotations options like the following:
-        /// <example>
-        /// new Dictionary&lt;string, int&gt; {
-        ///     {"x", 114},
-        ///     {"y", 198},
-        ///     {"duration", 5},
-        ///     {"radius", 3},
-        ///     {"rotation", 220},
-        ///     {"touchCount", 2}
-        /// }
-        /// </example>
-        /// </param>
-        public void Rotate(Dictionary<string, int> opts)
```

Comment:
this is actually a breaking change.
Not sure about the policy for this driver, but in java and python we mark them as deprecated first and them remove in the next major version

Reply:
We are in the RC stages anyway, and as you can see people can't use it anyway
so I don't see any downside